src/resources/views/buttons/delete.blade.php

```diff
@@ -1,3 +1,56 @@
 @if ($crud->hasAccess('delete'))
-	<a href="{{ url($crud->route.'/'.$entry->getKey()) }}" class="btn btn-xs btn-default" data-button-type="delete"><i class="fa fa-trash"></i> {{ trans('backpack::crud.delete') }}</a>
-@endif
\ No newline at end of file
+	<a href="javascript:void(0)" onclick="deleteEntry(this)" data-route="{{ url($crud->route.'/'.$entry->getKey()) }}" class="btn btn-xs btn-default" data-button-type="delete"><i class="fa fa-trash"></i> {{ trans('backpack::crud.delete') }}</a>
+@endif
+
```

Comment:
Hmm... that is entirely possible. We might have moved it there in the next versions @FzSalehi . Take note there 3.4 is no longer supported or maintained, there are A LOT of next versions:
<img width="302" alt="CleanShot 2023-04-19 at 12 40 36@2x" src="https://user-images.githubusercontent.com/1032474/233035288-90fb3636-81e7-45ab-ae6e-f9ca19b524b3.png">

In fact, we're working on polishing & releasing v6 as we speak 🤞

Reply:
yeah i just follwing upgrate guide 

> Hmm... that is entirely possible, thanks for the heads-up. We might have moved it there in the next versions @FzSalehi .
> 
> Take note there 3.4 is no longer supported or maintained, there are A LOT of next versions: <img alt="CleanShot 2023-04-19 at 12 40 36@2x" width="302" src="https://user-images.githubusercontent.com/1032474/233035288-90fb3636-81e7-45ab-ae6e-f9ca19b524b3.png">
> 
> In fact, we're working on polishing & releasing v6 as we speak crossed_fingers

yeah i just following upgrade guide .. just a little thought i had to shere .. it might be helpful. thanks for responding.